angular2/package.json

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "typescript-angular2",
+    "version": "1.0.0",
+    "description": "Angular 2/TypeScript demo",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/Microsoft/TypeScriptSamples.git"

```

Comment:
sure, didn't plan to check this in as is until the release is done. Intent was to suggest the structure and content of the sample